labgrid/driver/sshdriver.py

```diff
@@ -432,10 +439,23 @@ def get_status(self):
         """The SSHDriver is always connected, return 1"""
         return 1
 
-    def _scp_supports_explicit_sftp_mode(self):
+    def _ssh_version_check(self):
         version = subprocess.run(["ssh", "-V"], capture_output=True, text=True)
         version = re.match(r"^OpenSSH_(\d+)\.(\d+)", version.stderr)
-        major, minor = map(int, version.groups())
+        return map(int, version.groups())
+
+    def _ssh_supports_fork_with_stderr(self):
+        major, minor = self._ssh_version
+
+        # OpenSSH < 8.5 has bug about stderr close via -f
+        if major < 8 or (major == 8 and minor < 5):
+            return False
+        else:
+            return True
+        raise Exception(f"OpenSSH version {major}.{minor} does not support ssh mode")
```

Comment:
```suggestion
        return major < 8 or (major == 8 and minor < 5)
```
This should be sufficient.

Reply:
You're right, the other way around :)